src/game/engine/house.cpp

```diff
@@ -1221,6 +1287,22 @@ void HouseClass::Sell_Wall(cell_t cellnum)
     }
 }
 
+BuildingTypeClass *const HouseClass::Suggest_New_Building() const
```

Comment:
There is no point this returning a constant pointer, const BuildingTypeClass * is a possible return if the value returned isn't supposed to be altered however.